polaris-react/src/components/FullscreenBar/tests/FullscreenBar.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {mountWithApp} from 'tests/utilities';
+
+import {FullscreenBar} from '../FullscreenBar';
+
+describe('<FullscreenBar />', () => {
+  it('renders its children', () => {
+    const text = 'My App Info';
+    const bar = mountWithApp(
+      <FullscreenBar onAction={() => {}}>{text}</FullscreenBar>,
+    );
+
+    expect(bar).toContainReactText(text);
+  });
+
+  it('renders with no children', () => {
+    const bar = mountWithApp(<FullscreenBar onAction={() => {}} />);
+    const back = 'Back';
+
+    expect(bar).toContainReactText(back);
+  });
```

Comment:
This is not a great test

Reply:
Fair enough.  I was just trying to verify it would mount without any children.  I copied the `TopBar` test just called `mounts` now.

Reply:
Ended up just removing this test, as recommended in feedback.